src/aap_eda/core/management/commands/create_initial_data.py

```diff
@@ -24,6 +24,7 @@
 from django.db.models import Q
 
 from aap_eda.core import enums, models
+from aap_eda.core.models import utils
```

Comment:
@hsong-rh Would it be better to directly import the fuction instead of importing utils since that is such a generic name.
```
   from aap_eda.core.models.utils import get_default_organization
```

Reply:
@hsong-rh @mkanoor 
Not that my opinion matters (at least, anymore 😄) but... at one point I put out a post in the eda channel about avoiding "free-standing" (or whatever I called it) references.  The idea being to provide context for the origin of a reference making life somewhat easier when trying to hunt down all the usages of contents from a particular package (motivated by the effort to hunt down all redis-related usages when adding the connection retry decorator).  No one objected, but no one jumped up and down and said "Yes!" either.

Do with this what you will.

Reply:
I would prefer to import the module instead of the function, especially when we talk about "utils", it gives more context and makes easier to track usages in the future as joe pointed out.